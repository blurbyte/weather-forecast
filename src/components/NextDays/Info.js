import styled from 'styled-components';

import media from '../../styles/media';

const Info = styled.div`
  display: flex;
  margin: 0.4rem 0;

  ${media.tablet`
    margin-left: auto;
    width: calc(50% + 1.8rem);
  `};
`;

export default Info;
