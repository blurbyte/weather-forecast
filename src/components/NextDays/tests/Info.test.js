import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Info from '../Info';

test('renders correctly', () => {
  const component = renderer.create(<Info />);
  expect(component.toJSON()).toMatchSnapshot();
});
