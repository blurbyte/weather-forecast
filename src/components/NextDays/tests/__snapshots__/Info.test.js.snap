// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0.4rem 0;
}

@media (max-width:40em) {
  .c0 {
    margin-left: auto;
    width: calc(50% + 1.8rem);
  }
}

<div
  className="c0"
/>
`;
